tests/test_runners/test_raw.py

```diff
@@ -248,6 +250,63 @@ def test_record_tests(self):
                         'created_at': '2021-10-05T12:34:56',
                         'data': None,
                         'type': 'case',
+                        'metadata': None
+                    },
+                ],
+                "testRunner": "raw",
+                "group": "",
+                "noBuild": False,
+            })
+
+    @responses.activate
+    @mock.patch.dict(os.environ, {"LAUNCHABLE_TOKEN": CliTestCase.launchable_token})
+    def test_metadata_field(self):
```

Comment:
We can merge this test case to the `def test_record_tests` https://github.com/launchableinc/cli/pull/814/files#diff-ab0fb1f8c2f413fe3f1b4d7cc33b76f99d93d3cac06b353f285bb5546a3ac15fR262

Reply:
I tried it, but I got the following error: Thus, I separated `test_metadata_field` from `test_record_tests`
```
TypeError: '<' not supported between instances of 'list' and 'NoneType'
```